wolox-books/src/app/register-layout/register-layout.component.html

```diff
@@ -3,32 +3,15 @@
     <div class="login-inputs">
       <img class="logo-wolox-login" src="../../assets/LogoWolox-Original.png"  alt="Wolox Logotype" />
       <div class="form-container">
-        <form class="form-login" action="/index.html" id="form-login">
-          <div class="form-group">
-            <label class="form-label" for="email">Nombre</label>
-            <input class="default-rectangle form-input" type="text" id="email" name="name" />
-          </div>
-          <div class="form-group">
-            <label class="form-label" for="password">Apellido</label>
-            <input class="default-rectangle form-input" type="text" id="password" name="last-name" />
-          </div>
-          <div class="form-group">
-            <label class="form-label" for="password">Email</label>
-            <input class="default-rectangle form-input" type="text" id="password" name="email" />
-          </div>
-          <div class="form-group">
-            <label class="form-label" for="password">Password</label>
-            <input class="default-rectangle form-input" type="text" id="password" name="password" />
-          </div>
-          <div class="form-group">
-            <label class="form-label" for="password">Confirmación de Password</label>
-            <input class="default-rectangle form-input" type="text" id="password" name="password-confirm" />
-          </div>
+        <form class="form-login">
+          <div class="form-group" *ngFor="let field of fields">
+            <label class="form-label" [for]="field.id">{{field.name}}</label>
```

Comment:
Please use white space between open and close interpolation. example `{{ field.name }}`